### PR TITLE
[Localization][ko] Translate missed texts in filter bar.

### PR DESCRIPTION
**Pull request by EnochG1**
## What are the changes?
Add translation of missed text in filter bar.

## Why am I doing these changes?
New label texts are added.

## What did change?
* Add translations
* Fixed wrong message key (Case-wrong)

#### As-is
image
image

#### To-be
image
image

## How to test the changes?
Go to 'New Game'

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/ko/filter-bar.ts [hunk 1]
```diff
@@ -13,18 +13,18 @@ export const filterBar: SimpleTranslationEntries = {
...
-  "hiddenAbility": "Hidden Ability",
-  "hasHiddenAbility": "Hidden Ability - Yes",
-  "noHiddenAbility": "Hidden Ability - No",
-  "pokerus": "Pokerus",
-  "hasPokerus": "Pokerus - Yes",
-  "noPokerus": "Pokerus - No",
+  "hasWon": "클리어 완료",
+  "hasNotWon": "클리어 안함",
+  "hiddenAbility": "숨겨진 특성",
+  "hasHiddenAbility": "숨겨진 특성 있음",
```
Comment (EnochG1) [inline src/locales/ko/filter-bar.ts:23]:
```suggestion
  "hasHiddenAbility": "숨겨진 특성 보유",
```

**Review by sodaMelon — APPROVED** [review]
It's like it was translated by a Pokémon master. LGTM 😉✨

File: src/locales/ko/filter-bar.ts [hunk 2]
```diff
@@ -13,18 +13,18 @@ export const filterBar: SimpleTranslationEntries = {
   "passive": "패시브",
   "passiveUnlocked": "패시브 해금",
   "passiveLocked": "패시브 잠김",
-  "costReduction": "Cost Reduction",
-  "costReductionUnlocked": "Cost Reduction Unlocked",
-  "costReductionLocked": "Cost Reduction Locked",
+  "costReduction": "코스트 줄이기",
+  "costReductionUnlocked": "코스트 줄이기 함",
+  "costReductionLocked": "코스트 줄이기 안함",
```
Comment (EnochG1) [inline src/locales/ko/filter-bar.ts:18]:
```suggestion
  "costReductionUnlocked": "코스트 절감됨",
  "costReductionLocked": "코스트 절감 없음",
```

**Comment by KimJeongSun** [comment]
LGTM

**Review by Madmadness65 — APPROVED** [review]

**Pull request merged**